### DISCUSSION TITLE
Fixes for #113 and #114

### DISCUSSION THREAD

**Discussion by rcd09** [cogu/autosar#115] [General]
Hi all,

I have 3 lines of fixes for these 2 issues that are hindering my usage of 0.4.2.  Can I be made a collaborator to do these?  If not here are the changes:

In builder.py:
![Snap2](https://github.com/user-attachments/assets/38d0c734-3543-43a9-a19a-d82945fa0f5f)

In parser_base.py:
![Snap3](https://github.com/user-attachments/assets/b5e8dbc0-aa4b-4ecb-8234-b7a36e38b367)

**Comment by cogu** [comment]
Hi @rcd09,

I'm currently on vacation. Will take a look at it in about a week or so.
If you can't wait that long you're welcome to post a pull request.

**Comment by rcd09** [comment]
Hi cogu - I don't have permission to create a branch as far as I can see.  But then I use GitLab at work so might not see where the 'create branch' or 'create pull request' links are hiding.

**Comment by cogu** [comment]
On GitHub you begin by forking the repo and then you work on the fork. Then there should be options for creating a pull request upstream (my repo). I'm sure you can find a guide somewhere online for the exact steps.

A branch for v0.4 maintenance already exists so make sure to base or changes relative to that branch.

I admit it's a lot of steps for such a small change so if you want I can commit the fix on 0.4 branch once I have time.
